src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -235,7 +235,7 @@ object Instance {
       Condition.Unknown,
 
       //From here on all tasks are only in one of the following states
-      Condition.Created,
+      Condition.Provisioned,
```

Comment:
Why is `Scheduled` not in the hierarchy?

Reply:
I don't know, probably we forgot to add it there.